packages/context/src/ui5-yaml.ts

```diff
@@ -30,35 +21,35 @@ export async function initializeUI5YamlData(
   const readUI5YamlPromises = map(ui5YamlDocuments, async (ui5YamlDoc) => {
     const response = await readUI5YamlFile(ui5YamlDoc);
 
-    // Parsing of ui5.yaml failed because the file is invalid
-    if (response !== "INVALID") {
-      ui5YamlData[ui5YamlDoc] = response;
+    if (response) {
+      cache.setYamlDetails(ui5YamlDoc, response);
+      console.info("ui5.yaml data initialized", { ui5YamlDoc });
     }
   });
 
-  getLogger().info("ui5.yaml data initialized", { ui5YamlDocuments });
+  console.info("list of ui5.yaml files", { ui5YamlDocuments });
```

Comment:
We probably should keep the logger usage, because it is used to control log level via user setting.

Reply:
Thanks. Done